description: output instruction to mark the stack as non-executable
author: Michael Gilbert <mgilbert@debian.org>

--- a/tools/winebuild/import.c
+++ b/tools/winebuild/import.c
@@ -631,6 +631,7 @@ static char *create_undef_symbols_file(
     }
     for (j = 0; j < extra_ld_symbols.count; j++)
         output( "\t%s %s\n", get_asm_ptr_keyword(), asm_name(extra_ld_symbols.str[j]) );
+    output_gnu_stack_note();
     fclose( output_file );
 
     obj_file = get_temp_file_name( output_file_name, ".o" );
